riotfile.py

```diff
@@ -1517,7 +1517,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                 ),
                 Venv(
                     # fastapi added support for Python 3.11 in 0.86.0
-                    pys=select_pys(min_version="3.11"),
+                    pys="3.11",
```

Comment:
Why have you removed tests for Python 3.12? As far as I know it's still important to support Fastapi under 3.12 if we can.

Reply:
Fixed